truss/cli/cli.py

```diff
@@ -1001,24 +1006,27 @@ def view_training(
     if not remote:
         remote = remote_cli.inquire_remote_name()
 
-    if job_id and not project_id:
-        raise click.UsageError(
-            "Project ID is required when specifying a job ID. To see all projects, use `truss train view`"
-        )
-
     remote_provider: BasetenRemote = cast(
         BasetenRemote, RemoteFactory.create(remote=remote)
     )
 
-    if project_id:
-        if job_id:
+    if job_id:
```

Comment:
Given that our BE endpoint supports job_id / project_id, could we just hit /search with the user input instead of these conditions? Given that we know at least one was provided, else we default to the list projects code below